pins/rsconnect/html/index.html

```diff
@@ -50,15 +50,27 @@ <h3>{{pin_name}}</h3>
     <h3>Python Code</h3>
 
     <pre id="pin-python" class="pin-code"><code class="python">from pins import board_connect
+from dotenv import load_dotenv
+load_dotenv()
+
 board = {{board_deparse}}
 board.pin_read("{{pin_name}}")</code></pre>
 
     <script type="text/javascript">
       hljs.registerLanguage("python", highlight_python);
+      hljs.registerLanguage("r", highlight_r);
       hljs.initHighlightingOnLoad();
     </script>
     </section>
 
+    <section style="{{ show_r_style }}">
+      <h3>R Code</h3>
+        <pre id="pin-r" class="pin-code"><code class="r">library(pins)
+
+  board <- {{board_deparse}}
+  pin_read(board, "{{pin_name}}")</code></pre>
```

Comment:
```suggestion
board <- {{board_deparse}}
pin_read(board, "{{pin_name}}")</code></pre>
```

Reply:
We can't use the same `board_deparse` object here, right? It needs to be something R specific? I believe it should be `"board_connect(auth = "envvar")"`.

Reply:
ah, yep! if we wanted, we could deparse to be `board_connect()`, but the preview adds the raw `server_url` for Python. I've changed it to `board_connect(auth = "envvar")`